sci-libs/torchvision/files/torchvision-include-dir.patch

```diff
@@ -0,0 +1,15 @@
+diff -Naur a/setup.py b/setup.py
+--- a/setup.py	2020-03-07 14:28:12.935708034 -0300
++++ b/setup.py	2020-03-07 15:07:42.258164580 -0300
+@@ -125,7 +125,10 @@
+ 
+     sources = [os.path.join(extensions_dir, s) for s in sources]
+ 
+-    include_dirs = [extensions_dir]
++    c_include_dir = os.path.join('/usr/include','torch','csrc','api','include')
++    c_include_local_dir = os.path.join('/usr/local/include','torch','csrc','api','include')
++
```

Comment:
1. `/usr/include/include`?  What is this dir?`/usr/include/include`?  What is this dir?

2. Is /usr/local/include useful for us?

3. hard coding `/usr/include` breaks on Gentoo Prefix.  How about symlinking headers from `EPREFIX`/usr/include into `extensions_dir`?

Reply:
This is what I got after pytorch installed the include dirs in other location rather then default. Without that, the ebuild fails at detecting the include library, so I couldn't find another way to do so.